## Description: Removes extra spaces around the font in window titles
## Origin/Author: 0x2493
## Bug: https://github.com/ashinkarov/i3-extras
--- a/src/render.c
+++ b/src/render.c
@@ -25,10 +25,7 @@ static void render_con_dockarea(Con *con
  * Returns the height for the decorations
  */
 int render_deco_height(void) {
-    int deco_height = config.font.height + 4;
-    if (config.font.height & 0x01)
-        ++deco_height;
-    return deco_height;
+    return config.font.height + 2;
 }
 
 /*
